src/content/docs/guides/game design/godot/basics.md

```diff
@@ -0,0 +1,182 @@
+---
+title: Basics
+description: This is a page in my Starlight-powered site
+---
+
+[Godot](https://godotengine.org/) is a free and open source cross-platform game engine that supports game design for 2D and 3D games. It is a great tool for beginners and experts alike. It is also a great tool for teaching game design and programming. This guide will help you get started with Godot and teach you how to make a simple 2D game.
+
+This guide is up-to-date with Godot 4.1.1 stable official release. 
+# Getting started
+There are several options for developing with Godot. The recommended way is to download the latest version from the [downloads page](https://godotengine.org/download/) for Windows/macOS/Linux. It will be what we use for this guide.
+
+- Run the engine in a web browser with the [Godot Web Editor (v4.1.1)](https://editor.godotengine.org/releases/4.1.1.stable). This is the easiest way to get started, but it has [documented limitations](https://docs.godotengine.org/en/stable/getting_started/editor/web_editor.html#limitations) and is not recommended for large projects.
+- Download the android app from the [Google Play Store](https://play.google.com/store/apps/details?id=org.godotengine.editor.v4), while functional it is in early access and may not be stable. It is also difficult to use on phones, and tablets are still difficult without a keyboard and mouse.
```

Comment:
If it's in beta I would remove it from this

Reply:
It's in early access but it's totally available on the app store, I've used it and it just seems like a perfectly working version of godot